src/main/java/org/sonarsource/sonarlint/ls/SonarLintLanguageServer.java

```diff
@@ -509,4 +521,43 @@ public CompletableFuture<GetServerPathForTokenGenerationResponse> getServerPathF
     return ServerPathProvider.getServerUrlForTokenGeneration(endpointParams, httpClientProvider.anonymous(), port, appName)
       .thenApply(GetServerPathForTokenGenerationResponse::new);
   }
+
+  @Override
+  public void openHotspotInBrowser(OpenHotspotParams params) {
+    var hotspotId = params.getHotspotId();
+    var fileUri = create(params.getFileUri());
+    var folderForFileOptional = workspaceFoldersManager.findFolderForFile(fileUri);
+    if (folderForFileOptional.isEmpty()) {
+      lsLogOutput.error("Can't find workspace folder for file "
```

Comment:
In terms of UX, the user could click on the link and nothing would happen except the trace in the logs. Should this be a message for the client to display a notification ?